src/txn/txn.c

```diff
@@ -2422,6 +2422,12 @@ __wt_txn_global_shutdown(WT_SESSION_IMPL *session, const char **cfg)
               "[SHUTDOWN_INIT] performing shutdown rollback to stable, stable_timestamp=%s",
               __wt_timestamp_to_string(conn->txn_global.stable_timestamp, ts_string));
             WT_TRET(conn->rts->rollback_to_stable(session, cfg, true));
+
+            if (ret != 0)
+                __wt_verbose_notice(session, WT_VERB_RTS,
+                  WT_RTS_VERB_TAG_SHUTDOWN_RTS
+                  "performing shutdown rollback to stable failed with code %s",
```

Comment:
This won't quite match the regex because the equals sign is missing.

Reply:
[59e896b](https://github.com/wiredtiger/wiredtiger/pull/8933/commits/59e896b0b7b2c47938d502926e61165bba974949)